src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Configuration.targets

```diff
@@ -59,6 +59,15 @@ Copyright (c) .NET Foundation. All rights reserved.
     <RazorDefaultConfiguration Condition="'$(_TargetingNETCoreApp30OrLater)' == 'true'">MVC-3.0</RazorDefaultConfiguration>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(RazorDefaultComponentConfiguration)'==''">
+    <RazorDefaultComponentConfiguration Condition="'$(_TargetingNETCoreApp30OrLater)' == 'true'">Component-1.0</RazorDefaultComponentConfiguration>
```

Comment:
If this is temporary for while we add support for Razor - I guess I'd prefer to do it differently. This is going to run components an regular views in two 'batches' which isn't super integrated.

Reply:
This was removed